apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js

```diff
@@ -100,13 +99,9 @@ describe('CircuitPlaygroundBoard', () => {
   );
 
   describe(`connect()`, () => {
-    // Remove these lines when maker-captouch is on by default.
-    before(() => experiments.setEnabled('maker-captouch', true));
-    after(() => experiments.setEnabled('maker-captouch', false));
-
     it('initializes a set of components', () => {
       return board.connect().then(() => {
-        expect(Object.keys(board.prewiredComponents_)).to.have.length(23);
+        expect(Object.keys(board.prewiredComponents_)).to.have.length(16);
```

Comment:
What does the magic number mean? Could we bump this to a constant somewhere else?

Reply:
Added comment.